explainers/hidden-content-explainer.md

```diff
@@ -271,15 +233,83 @@ only benefit I see to adding support to `window.find` would be to make it easier
 to add WPT tests for beforematch, since we cant have WPTs for find-in-page.
 However, that would also require speccing `window.find`.
 
+## Backwards compatibility with the `hidden` attribute
+
+If a browser which doesn't implement this feature renders a `hidden=until-found`
+element, it will still apply the `display:none` style from the `hidden`
+attribute, which will still make the element hidden like the
+`content-visiblity:hidden` style that `hidden=until-found` is supposed to apply.
+This means that hopefully websites won't need to specially consider support for
+this feature between browsers.
+
+However, `content-visibility:hidden` will still allow some parts of the
+`hidden=until-found` element to be rendered: the [generated
+box](https://drafts.csswg.org/css2/#visibility) of the element.
+
+As a mitigation for this, the following CSS should be included in the user agent
```

Comment:
"To allow compatibility with hidden attribute, and to make adoption straight forward, the user-agent stylesheet must also include the following:"

I know this isn't a spec, but as a general comment in spec languages "should" typically means an optional thing which user-agents may omit (csswg tends to follow this: https://datatracker.ietf.org/doc/html/rfc2119, not sure about whatwg/html)

Reply:
done